akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolConductor.scala

```diff
@@ -153,6 +164,27 @@ private object PoolConductor {
       override def preStart(): Unit = {
         pull(ctxIn)
         pull(slotIn)
+
+        startMinConnections()
+      }
+
+      def startMinConnections(): Unit =
+        (0 until slotSettings.minSlots).foreach { connect }
+
+      // pool ordering connection doesn't change the status of a connection to Idle,
+      // receiving confirmation from the Slot does. See: handler for "slotIn"
+      def connect(index: Int): Unit =
+        emit(out, SlotShouldConnectCommand(index))
+
+      private def reconnectIfNeeded(): Unit = {
+        val connectedNum = slotStates.count(_ != Unconnected)
+        if (connectedNum < slotSettings.minSlots) {
+          val unconnected = slotStates
+            .zipWithIndex
+            .filter(_._1 == Unconnected)
+            .take(slotSettings.minSlots - connectedNum)
+          unconnected.map(_._2).map(connect)
+        }
```

Comment:
See https://github.com/2beaucoup/akka/commit/1c89d705db5fc9ada290f4b0b7bedbe7101c066b#diff-0dc3503e2c7e4c93cd4b99e13e236cacR143 how this can be simplified.